### ISSUE TITLE
SafeLoader NameError with Docker [latest]

### ISSUE THREAD

**Issue by skchronicles** [ewels/MultiQC#917]
**Description of bug:**
Getting the following error message when using the `latest` docker version of MultiQC with python2:
`NameError: name 'SafeLoader' is not defined`

**MultiQC Error log:**
```
Downloading files using 4 threadssha256:ead11ecb983: 2.53 MB/2.53 MB
sha256:1c63dfc85c0: 2.09 MB/2.09 MB
sha256:6ae821421a7: 22.5 MB/22.5 MB
sha256:d1ef2eb755b: 16.2 MB/17.1 MB
sha256:49850c10032: 8.48 MB/79.1 MB
sha256:7373616471e: 6.61 MB/82.6 MB
sha256:d1ef2eb755b: 17.1 MB/17.1 MB
sha256:49850c10032: 36 MB/79.1 MB
sha256:7373616471e: 16.9 MB/82.6 MB
sha256:49850c10032: 79.1 MB/79.1 MB
sha256:7373616471e: 46.7 MB/82.6 MB
sha256:7373616471e: 81.5 MB/82.6 MB
Image Repo	ewels/multiqc
Image Tag	latest
Image Size	42M
Image ID	sha256:49850c10032dad5ec3ef363c02ab8c8c9b5a8e4ed4e502144ca559b3d71a04c7
Last Updated	2019-03-03T15:53:00.486358716Z (0d 23h ago)
Registry	registry-1.docker.io
Default CMD	["python2"]
proot warning: can't sanitize binding "/var/run/dbus/system_bus_socket": No such file or directory
Traceback (most recent call last):
  File "/usr/local/bin/multiqc", line 32, in <module>
    from multiqc import __version__
  File "/usr/local/lib/python2.7/site-packages/multiqc/__init__.py", line 2, in <module>
    from multiqc.utils import config
  File "/usr/local/lib/python2.7/site-packages/multiqc/utils/config.py", line 45, in <module>
    configs = yaml.load(f, Loader=SafeLoader)
NameError: name 'SafeLoader' is not defined
```  

**MultiQC run details (please complete the following):**
 - Command used to run MultiQC: [eg. `multiqc --help`]
 - MultiQC Version: [eg. MultiQC dockerhub:latest]
 - Operating System: [eg. Cent)O7]
 - Python Version: [eg. Python 2.7]
 - Method of MultiQC installation: [eg. docker (dockerhub: tag == latest)]

**Comment by ewels** [comment]
Hi @skchronicles,

I think you've been very unlucky and pulled a copy of the development code some time between `16:36:13` and `16:53:51` yesterday when I added that line. It broke the tests and I reverted it then fixed it pretty quickly. You can see that this line now reads differently:

https://github.com/ewels/MultiQC/blob/23c42f91f4b630e9ef875dcba165bfc9d911a1e9/multiqc/utils/config.py#L45

Could you try grabbing the latest version of the development code and seeing if it works now?

Cheers,

Phil

**Comment by ewels** [comment]
Ah wait, just noticed that you're using the docker version. Hah, that's really unfortunate... I switched the docker hub auto-build behaviour yesterday so that in the future `:latest` will instead have the most recent release and `:dev` will have the development code. I guess that the last build I did before I made that switch was with the broken code though. I'll push a new version of the `:latest` tag now.

Phil
